test/serverSpec.js

```diff
@@ -0,0 +1,27 @@
+const assert = require('assert');
+const request = require('request');
+
+describe('databse', function() {
```

Comment:
@Pegaiur database is spelled incorrectly

Reply:
thanks @bnguyen2